packages/wrangler/src/versions/deploy.ts

```diff
@@ -44,61 +41,73 @@ const EPSILON = 0.001; // used to avoid floating-point errors. Comparions to a v
 const BLANK_INPUT = "-"; // To be used where optional user-input is displayed and the value is nullish
 const ZERO_WIDTH_SPACE = "\u200B"; // Some log lines get trimmed and so, to indent, the line is prefixed with a zero-width space
 
-export type VersionsDeployArgs = StrictYargsOptionsToInterface<
-	typeof versionsDeployOptions
->;
+export type VersionsDeployArgs = typeof command.args;
 
 type OptionalPercentage = number | null; // null means automatically assign (evenly distribute remaining traffic)
 
-export function versionsDeployOptions(yargs: CommonYargsArgv) {
-	return yargs
-		.option("name", {
+const command = defineCommand({
+	command: "wrangler versions deploy",
```

Comment:
Do we have a plan as to how sub-commands are defined? Is defineCommand composable (or could it be)?

Reply:
All commands + subcommands are expected to be defined fully and independently. This example is a subcommand itself.